client/card-readers/settings/sections/contacts-details.js

```diff
@@ -5,7 +5,7 @@
 import React, { useEffect, useState } from 'react';
 import { __ } from '@wordpress/i18n';
 import { TextControl, Notice, BaseControl } from '@wordpress/components';
-import PhoneNumberInput from 'settings/phone-input';
+const PhoneNumberInput = React.lazy( () => import( 'settings/phone-input' ) );
```

Comment:
We can take one step further and name the new chunk more human friendly:

```js
const PhoneNumberInput = React.lazy( () =>
	import(
		/* webpackChunkName: "settings-phone-input" */ 'settings/phone-input'
	)
);
```

![Screenshot from 2022-11-28 12-46-16](https://user-images.githubusercontent.com/1577185/204270281-eb52e828-c349-4a75-a9d1-0e07292a9852.png)

Reply:
+1 to Vlad's suggestion.